exceptional_operations.md

```diff
@@ -17,9 +17,9 @@ By default, Uptane does not allow updates to be rolled back and enforces this ac
 
 **Figure 1.** *Uptane prevents rollback attacks by rejecting older: (1) metadata files, and / or (2) images.*
 
-There are at least two ways to allow rollbacks, each with different advantages and disadvantages.
+There are at least two ways to allow updates to roll backs, each with different advantages and disadvantages.
```

Comment:
"rollbacks" seems better to me than "roll backs". I think the original phrase here was fine, but I could see room for improving the clarity.

Reply:
On second thought I'm going to keep this as is.  I think it reads better
```suggestion
There are at least two ways to allow rollbacks, each with different advantages and disadvantages.
```